core-web/libs/sdk/experiments/project.json

```diff
@@ -5,14 +5,32 @@
     "projectType": "library",
     "targets": {
         "build": {
-            "executor": "@nx/vite:build",
+            "executor": "@nrwl/rollup:rollup",
             "outputs": ["{options.outputPath}"],
             "options": {
                 "outputPath": "dist/libs/sdk/experiments",
-                "main": "libs/sdk/experiments/src/lib/standalone.ts",
                 "tsConfig": "libs/sdk/experiments/tsconfig.lib.json",
                 "project": "libs/sdk/experiments/package.json",
-                "assets": ["libs/sdk/experiments/README.md", "libs/sdk/experiments/package.json"]
+                "entryFile": "libs/sdk/experiments/src/index.ts",
+                "external": ["react/jsx-runtime"],
+                "rollupConfig": "@nrwl/react/plugins/bundle-rollup",
+                "compiler": "babel",
+                "assets": [
+                    {
+                        "glob": "libs/sdk/experiments/README.md",
+                        "input": ".",
+                        "output": "."
+                    }
+                ]
+            }
+        },
+        "build:iife": {
```

Comment:
I left the config of VITE to generate the IIFE.

Reply:
This command generates only one single javascript file?
Maybe we can use in SDK Editor - VTL
thoughts? cc @fmontes

Reply:
No is the best approach because we are using rollout. But after all this, I will try to do the same with rollout. But by now was the only way to use the class standalone for the HTML demo